src/VisualStudio/CSharp/Test/PersistentStorage/AbstractPersistentStorageTests.cs

```diff
@@ -52,7 +51,6 @@ private void CleanUpPersistentFolder()
         {
         }
 
-        [Fact]
```

Comment:
This didn't work when i tried that out myself.  Though that may be an artifact of the test runner i was using.